CMakeLists.txt

```diff
@@ -118,7 +118,15 @@ else ()
     -Wuninitialized
   )
 
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wold-style-cast")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wold-style-cast")
```

Comment:
Why this?  Should we fix the instances instead?

Reply:
Sorry, I'm not sure I get what you mean; `-Wold-style-cast` is C++ only thus I think it could not be absorbed into `add_definitions` (as we're on CMake3, perhaps they meant `add_compile_options`) above. It seems it was introduced in #320.

Reply:
Sorry I mis-read the diff.  Through you were adding the `-W` flag.